doc/Language/pragmas.pod6

```diff
@@ -41,25 +43,35 @@ and turns on its features if they are optional.
 Since these pragmas turn on the compiler version, they should be the
 first statement in the file (preceeding comments and Pod are fine).
 
-=item X<B<MONKEY-GUTS>|MONKEY-GUTS>
+=head2 MONKEY-GUTS
```

Comment:
Please carry the indexing (X<>) over to this new version too.

Reply:
Doesn't `=head` stuff get automatically indexed? Also, I recall there being a problem with `X<>` not supported in `=head` and we had to put them above `=head` instead

Reply:
It does only in /Types, if I remember correctly. If it's in a `head` it does strange stuff (creating categories). Yep, it should be over the header.